frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx

```diff
@@ -354,15 +406,22 @@ const createDefaultArgs = (
     ...args,
   };
 };
-
-function getLastPopover() {
+function getLastPopover(wrapped?: true): ReturnType<typeof within>;
+function getLastPopover(wrapped: false): HTMLElement;
```

Comment:
Woah, nice use of overloading here! I always forgot you can do this to map the input to output types ❤️

Reply:
Actually I couldn't figure out the return type `ReturnType<typeof within>` is wrong, and I ended up creating 2 functions instead for simplicity and let TypeScript infers the return type.

Reply:
That said, I if we could figure out what's the type of `within(element)`, then we could come back to use this overloading function again, but I don't want to waste too much time here.